src/privileged/FirefoxMonitor.jsm

```diff
@@ -260,6 +291,9 @@ this.FirefoxMonitor = {
           getFormattedString: (aKey, args) => {
             return this.getFormattedString(aKey, args);
           },
+          getFirefoxMonitorURL: (aSiteName) => {
+            return `${this.FirefoxMonitorURL}/?breach=${aSiteName}&utm_source=firefox&utm_medium=popup`;
```

Comment:
Doubt this matters, but url-encode the `aSiteName` here?

Reply:
Done, with r=pdehaan ;)